hnsecure/src/main/java/ca/bc/gov/hlth/hnsecure/routes/HIBCRoute.java

```diff
@@ -21,12 +21,15 @@
 
 public class HIBCRoute extends BaseRoute {
 
+	private static final String DIRECT_AUDIT = "direct:audit";
+	private static final String DIRECT_HIBC_MQ = "direct:hibcMQ";
+
 	@Override
 	public void configure() throws Exception {
 
-		String hibcHttpUrl = String.format(properties.getValue(ApplicationProperty.HIBC_HTTP_URI) + "?bridgeEndpoint=true");
+		String hibcHttpUrl = String.format(properties.getValue(ApplicationProperty.HIBC_HTTP_URI) + "?bridgeEndpoint=true");//NOSONAR
```

Comment:
is this comment to ignore sonar code smell?

Reply:
Yes.

Reply:
Will be better if we can add Sonar ignore errors at one place. If not trivial, adding a comment of what sonar error we are ignoring.